docs/sphinx/examples/python/tutorials/readout_error_mitigation.ipynb

```diff
@@ -0,0 +1,845 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5b2ac00e-c7fc-45cc-888e-8c26dabf80d2",
+   "metadata": {},
+   "source": [
+    "# Readout Error Mitigation\n",
+    "\n",
+    "Readout errors are caused by imperfect qubit measurement, and are a common source of error in quantum computing. Properly modeling these errors in simulation can give the user tools to better understand how these errors and how to mitigate them when running on actual quantum devices.\n",
+    "\n",
+    "Readout errors can be mitigated with a confusion matrix $A$. It is a square matrix of size $2^n \\times 2^n$ and tells about the probability of observing the a state $|y\\rangle$ given the true state $|x\\rangle$. The confusion matrix characterizes the readout error of the device and is circuit independent. Once $A$ is estimated, we can compute its psuedoinverse $A^+$ which can be applied to the noisy probability distribution $p_{\\text{noisy}}$ to obtain an adjusted quasi-probability distribution \n",
```

Comment:
Typo: "psuedoinverse"?

Reply:
Yes. I think it should be "pseudoinverse". The [mitiq](https://mitiq.readthedocs.io/en/stable/guide/rem-5-theory.html) docs also mistyped it.